README.md

```diff
@@ -4,33 +4,33 @@
-    * [1.4. Tips to harden your image](https://github.com/wso2/container-guide)
-    * [1.5. Tips on Tagging](https://github.com/wso2/container-guide)
-    * [1.6. Vulnerability scanning and container security check](https://github.com/wso2/container-guide)
-    * [1.7. Image signing and Verification](https://github.com/wso2/container-guide)
+- ### [1. Building a container image](./build/Build.md)
+    * [1.1. Why should you choose Docker as your container runtime ?](./build/Build.md#why-docker)
+    * [1.2. Using WSO2 images](./build/Build.md#using-wso2-images)
+    * [1.3. Building from source](./build/Build.md#building-from-source)
+    * [1.4. Tips to harden your image](./build/Build.md#tips-to-harden-your-image)
+    * [1.5. Tips on Tagging](./build/Build.md#tips-on-tagging)
+    * [1.6. Vulnerability scanning and container security check](./build/Build.md#scanning)
+    * [1.7. Image signing and Verification](./build/Build.md#signing)
 
-- ### [2. Pushing your image to a container registry](https://github.com/wso2/container-guide)
-    * [2.1. Why should you have your own private registry ?](https://github.com/wso2/container-guide)
-    * [2.2. How to secure your private registry ?](https://github.com/wso2/container-guide)
-    * [2.3. Scanning and Auditing](https://github.com/wso2/container-guide) 
+- ### [2. Pushing your image to a container registry](./push/Push.md)
+    * [2.1. Why should you have your own private registry ?](./push/Push.md#why-private-registry)
+    * [2.2. How to secure your private registry ?](./push/Push.md#how-to-secure)
+    * [2.3. Scanning and Auditing](./push/Push.md#scanning-auditing) 
 
-- ### [3. Deploying into production](https://github.com/wso2/container-guide)
-    * [3.1. Why do you need a container orchestrator ?](https://github.com/wso2/container-guide)
-    * [3.2. Why should you choose Kubernetes ?](https://github.com/wso2/container-guide)
-    * [3.3. Kubernetes Vs. OpenShift ?](https://github.com/wso2/container-guide)
-    * [3.4. Helm as package manager](https://github.com/wso2/container-guide) 
-    * [3.5. Supporting Infrastructure](https://github.com/wso2/container-guide)
-    * [3.6. Resource Allocation](https://github.com/wso2/container-guide)
-    * [3.7. Deployment definition tips](https://github.com/wso2/container-guide) 
-    * [3.8. Managing configurations](https://github.com/wso2/container-guide)  
-    * [3.9. Managing keystores, certificates and other artifacts](https://github.com/wso2/container-guide/blob/master/deploy/Managing_Keystores_And_Truststores.md)
-    * [3.10. Managing databases](https://github.com/wso2/container-guide)
-    * [3.11. Production deployment tips](https://github.com/wso2/container-guide)
-    * [3.12. Managing security](https://github.com/wso2/container-guide)
+- ### [3. Deploying into production](./deploy/Deploy.md)
+    * [3.1. Why do you need a container orchestrator ?](./deploy/Deploy.md)
```

Comment:
@DilanUA we named the files with the participle form of the relevant verb? Shall we follow the same format in the these new chapters as well?

Also, we have linked to the relevant directory in the main chapter link (e.g., `[3. Deploying into production](./deploy)`) since there could be multiple files under the chapter.

Reply:
@chirangaalwis done